### DISCUSSION TITLE
Need Help Compiling for ESP32C3 NO USB

### DISCUSSION THREAD

**Discussion by JustDr00py** [micropython/micropython#15113] [ESP32]
Im having issues with the current ESP32C3 Download it has USB enabled on pin 18 and 19 and I cant seem to compile it without USB. I tried to change it in sdkconfig but when i compile it it just errors out.

**Comment by mattytrentini** [comment]
For reference, @JustDr00py is using [this board](https://raspberrypi.web.za/shop/modules/esp-c3-12f-ac-powered-single-relay-board/); it's micro is housed in an Ai.Thinker module that is pin-compatible with the ESP8266 ESP-12F but is actually powered by an ESP32-C3.
